lib/Modules/OVSCharmUtils/OVSCharmUtils.psm1

```diff
@@ -39,6 +39,19 @@ $OVS_SERVICES = @{
 }
 
 
+function Get-OVSBridgeName {
+    $netType = Get-NetType
+    $vmSwitchName = Get-JujuVMSwitchName
+    if ($netType -eq 'ovs') {
+        $cfg = Get-JujuCharmConfig
+        $managementOS = $cfg['vmswitch-management']
```

Comment:
I think we can have the same bridge name regardless of whether or not management OS is enabled on it.